spec/plans/provision.fmf

```diff
@@ -203,3 +204,22 @@ example: |
                     model: 67
               - cpu:
                     model: 69
+
+/multi-host:
+    summary: Multi-host testing specification
+
+    description: |
+        As a part of the provision step it is possible to assign
+        a role using ``role`` parameter for multi-host testing.
+        A role can be arbitrary identifier, for example there can
+        be server and client roles for server-client multi-host
+        provisioning.
+
+    example: |
+        provision:
+            - name: server provisioning
+              how: local
+              role: server
+            - name: client provisioning
+              how: local
```

Comment:
The `local` method means running on the localhost, I guess this could be a bit misleading. What about using `virtual` instead?

Reply:
right, having two machines on the same local does not make sense

Reply:
Should be covered by the latest changes.